code/modules/html_interface/map/adv_sec/adv_camera.dm

```diff
@@ -248,7 +248,9 @@ var/global/datum/interactive_map/camera/adv_camera = new
 				A.eyeobj.forceMove(get_turf(cam))
 				A.client.eye = A.eyeobj
 			else
+				var/mob/M = usr
```

Comment:
`usr` is a mob by default, I'm not sure what this cast does.

Reply:
I assumed it might not be the case